plugin/kubernetes/kubernetes.go

```diff
@@ -502,10 +501,11 @@ func (k *Kubernetes) findServices(r recordRequest, zone string) (services []msg.
 
 			err = nil
 
-			s := msg.Service{Host: svc.ClusterIP, Port: int(p.Port), TTL: k.ttl}
-			s.Key = strings.Join([]string{zonePath, Svc, svc.Namespace, svc.Name}, "/")
-
-			services = append(services, s)
+			for _, ip := range svc.ClusterIPs {
```

Comment:
If the dual stack feature is not enabled, do we need to preserve signal ClusterIP processing？

Reply:
If dual stack is not enabled, ClusterIPs will contain the single ClusterIP as the first element.